README.md

```diff
@@ -1 +1,244 @@
-# morse-code
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="app-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] You are meant to remove this Microverse logo as it is a placeholder that came with the template. You can add your logo if you have any or leave out the logo entirely. :small_airplane: :wave: 

- - Screenshot:
![Screenshot from 2023-08-30 12-03-10](https://github.com/RileyManda/morse-code/assets/107632780/fd698154-9466-4d5a-a3f2-8dfec9d20a47)

Reply:
Noted with Thanks @V-Blaze